Modules/Quotas/src/Quotas.Infrastructure.Database.Postgres/Migrations/20230421155550_Init_Quotas.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Quotas.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class Init_Quotas : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Tier",
```

Comment:
I'm confused. Why is this table called "Tier" instead of "Tiers"? According to the naming conventions, the table is always called like the corresponding DbSet on the DbContext.

That should be fixed. And please don't edit the migrations directly.